R/zchunk_LB125.LC_tot.R

```diff
@@ -13,8 +13,7 @@
 #' @importFrom dplyr filter mutate select
 #' @importFrom tidyr gather spread
 #' @author YourInitials CurrentMonthName 2017
```

Comment:
You need all of your header information, describing what the chunk does and a little bit of how, filled out.